Formula/asn.rb

```diff
@@ -0,0 +1,22 @@
+class Asn < Formula
+  desc "Organization lookup and server tool (ASN / IPv4 / IPv6 / Prefix / AS Path)"
+  homepage "https://github.com/nitefood/asn"
+  url "https://raw.githubusercontent.com/nitefood/asn/64e92533f57e9261d1d6588b4714a4183fc3c304/asn"
```

Comment:
This needs to be a versioned tarball. See https://docs.brew.sh/Acceptable-Formulae#stable-versions.